README.md

```diff
@@ -15,6 +15,10 @@ To build the latest version of Cantaloupe from source, use the following:
 
     docker build --build-arg CANTALOUPE_VERSION=latest -t cantaloupe .
 
+If the upstream Cantaloupe project is broken, you can also build a last known good commit. To do this, supply a `LATEST_SAFE_COMMIT` argument:
```

Comment:
I suggest renaming LATEST_SAFE_COMMIT to "COMMIT_REF" and then document how to use it to check out the latest develop branch. _Put the power in the hands of the user_. We can utilize a commit ref for other purposes, such as referencing a release tag or a feature branch.

+1 with this change.

Reply:
New commit pushed with this change